src/components/Text.js

```diff
@@ -25,6 +25,8 @@ const Text = ({
   underline,
   display = 'inline-block',
   hyphenation = false,
+  // this should only be applied when children is of type string
```

Comment:
Could the comment or prop name be changed to make usage a little more clear?

We were (only momentarily) confused in review chat by 'being applied' requiring it to be set to false.

Reply:
Since it's a boolean, `showOrphans` or `displayOrphans` would be more suitable. Would this add enough clarity for you?

Reply:
Sounds great!

Reply:
Oops - edited... any of those. :)